sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/aio/_client.py

```diff
@@ -1,466 +1,99 @@
-    async def append_to_ledger(
-        self,
-        entry_contents: str,
-        *,
-        sub_ledger_id: Optional[str] = None,
-        wait_for_commit: bool = False,
-        **kwargs: Any,
-    ) -> AppendResult:
-        """Appends an entry to the Confidential Ledger.
-
-        :param entry_contents: Text to write to the ledger.
-        :type entry_contents: str
-        :param sub_ledger_id: Identifies the sub-ledger to append to, defaults to None. If none is
-            specified, the service will use the service-default sub-ledger id.
-        :type sub_ledger_id: Optional[str]
-        :param wait_for_commit: If True, this method will not return until the write is
-            durably saved to the ledger, defaults to False.
-        :type wait_for_commit: bool, defaults to False.
-        :return: Details about the write transaction.
-        :rtype: ~azure.confidentialledger.AppendResult
-        :raises: ~azure.core.exceptions.HttpResponseError
-        """
-
-        if entry_contents is None:
-            raise ValueError("entry_contents must not be None")
-
-        # pylint: disable=protected-access
-        result = await self._client.confidential_ledger.post_ledger_entry(
-            contents=entry_contents,
-            sub_ledger_id=sub_ledger_id,
-            cls=kwargs.pop("cls", AppendResult._from_pipeline_result),
-            **kwargs
+        self._client = AsyncPipelineClient(
+            base_url=_endpoint, config=self._config, **kwargs
         )
 
-        if wait_for_commit:
-            await self.wait_until_durable(result.transaction_id, **kwargs)
-
-        return result
-
-    @distributed_trace_async
-    async def create_or_update_user(
-        self, user_id: str, role: Union[str, LedgerUserRole], **kwargs: Any
-    ) -> LedgerUser:
-        """Creates a new Confidential Ledger user, or updates an existing one.
-
-        :param user_id: Identifies the user to delete. This should be an AAD object id or
-            certificate fingerprint.
-        :type user_id: str
-        :param role: Role to assigned to the user.
-        :type role: str or LedgerUserRole
-        :return: Details of the updated ledger user.
-        :rtype: ~azure.confidentialledger.LedgerUser
-        :raises: ~azure.core.exceptions.HttpResponseError
-        """
-
-        if user_id is None or role is None:
-            raise ValueError("user_id or role cannot be None")
-
-        result = await self._client.confidential_ledger.create_or_update_user(
-            user_id=user_id,
-            assigned_role=role.value if isinstance(role, LedgerUserRole) else role,
-            **kwargs
-        )
-        return LedgerUser(
-            user_id=result.user_id, role=LedgerUserRole(result.assigned_role)
+        self._serialize = Serializer()
+        self._deserialize = Deserializer()
+        self._serialize.client_side_validation = False
+        self.confidential_ledger = ConfidentialLedgerOperations(
```

Comment:
I didn't touch this since it was autogenerated. I agree it is verbose and unwieldy. Is the change I made in `_patch.py` what is expected? Roughly,

```python
class ConfidentialLedgerClient(GeneratedClient, ConfidentialLedgerOperations)
```

Reply:
(Screenshot of change for convenience)
![image](https://user-images.githubusercontent.com/9373058/174688459-fc9440d6-b324-468c-9eaa-6880a1c09404.png)